src/App.tsx

```diff
@@ -1,14 +1,68 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+const USER_ID = 10876;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<string>('all');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    };
+
+    fetchData();
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const completedTodos = todos.filter(todo => todo.completed);
+  const filteredTodos = {
+    all: todos,
+    active: activeTodos,
+    completed: completedTodos,
+  };
+  const visibleTodos:Todo[]
+    = filteredTodos[filterType as keyof typeof filteredTodos];
+
+  const messageDelete = document.querySelector('.delete');
```

Comment:
don't need to use document.querySelector, because in React you can get template elements with useRef